docs/excel/excel-add-ins-ranges-set-format.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Set the format of a range using the Excel JavaScript API
+description: 'Code samples that show how to use the Excel JavaScript API to set the format of a range.'
+ms.date: 04/02/2021
+localization_priority: Normal
+---
+
+# Set range format using the Excel JavaScript API
+
+This article provides examples that show how to set font color, fill color, and number format for cells in a range with the Excel JavaScript API. For the complete list of properties and methods that the `Range` object supports, see the [Excel.Range class](/javascript/api/excel/excel.range).
+
+[!include[Excel cells and ranges note](../includes/note-excel-cells-and-ranges.md)]
+
+## Set font color and fill color
+
+The following code sample sets the font color and fill color for cells in range **B2:E2**.
+
+```js
+Excel.run(function (context) {
+    var sheet = context.workbook.worksheets.getItem("Sample");
+
+    var range = sheet.getRange("B2:E2");
+    range.format.fill.color = "#4472C4";
+    range.format.font.color = "white";
+
+    return context.sync();
+}).catch(errorHandlerFunction);
+```
+
+### Data in range before font color and fill color are set
```

Comment:
The other articles used italics before the before-and-after images, but this one uses an H3. We should standardize this. I like an H3, but I'll defer to @lindalu-MSFT on which is preferable.

Reply:
I'll switch to all H3/H4 (depending on context), no italics.